svthalia/concrexit
# Thalia Pay for registrations

Closes #935

### Summary
New members can now indicate that they want to pay their membership fees with Thalia Pay. When a member is created from a registration, not only a `Member` but now also a `BankAccount` object will be created. Moreover, when accepting entries, those registrations will immediately go from `accepted` to `completed` and automatically a payment will be created.

### How to test
1. Go, as an anonymous user, to the member/benefactor registration page
2. Register (and choose to use a bank account)
3. Go through the review proces
4. Check that all messages that are sent are correct
5. Check that a BankAccount is created, a payment is added and the registration is completed automatically after accepting
6. Make sure that for non-thalia-pay registrations everything also keeps working
